Dependency nativescript-socket.io does not seem to work with nativescript-vue 2.0

Switching to nativescript-socketio did help me to get it compile and work. With nativescript-socket.io when building clean project from template with devtools enabled I get:
```
Failed to build plugin nativescript-socket.io :
Error: Command gradlew.bat failed with exit code 1 Error output:

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':lintClassPath'.
> Could not find lint-gradle-api.jar (com.android.tools.lint:lint-gradle-api:26.1.2).
  Searched in the following locations:
      https://jcenter.bintray.com/com/android/tools/lint/lint-gradle-api/26.1.2/lint-gradle-api-26.1.2.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/index.js
+++ b/index.js
@@ -45,6 +45,16 @@ function getServerIpAddress(host, port) {
   return `127.0.0.1:${port}`
 }
 
+// Wrap the toast message in a try, devtool still work without toaster
+const showToast = (message) => {
+  try {
+    const Toasty = require('nativescript-toasty').Toasty;
+    new Toasty(message).show();
+  } catch (error) {
+    console.log(error);
+  }
+}
+
 module.exports = function install(Vue, {debug = false, host = null, port = 8098} = {}) {
   const startApp = Vue.prototype.$start
 
@@ -52,12 +62,13 @@ module.exports = function install(Vue, {debug = false, host = null, port = 8098}
     const setupDevtools = () => {
       devtools.connect('ws://localhost', port, {
         app: this,
-        showToast: (message) => require('nativescript-toast').makeText(message).show(),
+        showToast,
         io() {
           const address = `http://${getServerIpAddress(host, port)}`
-          const SocketIO = require('nativescript-socket.io')
-          debug && SocketIO.enableDebug()
-          return SocketIO.connect(address)
+          const SocketIO = require('nativescript-socketio').SocketIO
+          let socketIO = new SocketIO(address, { debug: debug })
+          socketIO.connect()
+          return socketIO
         }
       })
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-vue-devtools",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "description": "A NativeScript-Vue plugin for connecting to the standalone vue-devtools",
   "main": "index.js",
   "scripts": {
@@ -27,8 +27,8 @@
   },
   "homepage": "https://github.com/nativescript-vue/nativescript-vue-devtools#readme",
   "peerDependencies": {
-    "nativescript-toast": "*",
-    "nativescript-socket.io": "*",
+    "nativescript-toasty": "*",
+    "nativescript-socketio": "*",
     "tns-core-modules": "*",
     "@vue/devtools": "*"
   }

